theorems/T000292.md

```diff
@@ -0,0 +1,16 @@
+---
+uid: T000292
+if:
+  and:
+    - P000002: true
+    - P000028: true
+    - P000136: true
+then:
+  P000052: true
+refs:
+  - mathse: 517411
+    name: in a topological space only finite subsets are compact sets

```

Comment:
I changed the title of the mathse post to the more readable "Is a space where only finite subsets are compact sets always discrete?" [here](https://math.stackexchange.com/questions/517411).